test/Feature/InsertSpec.hs

```diff
@@ -537,3 +537,13 @@ spec actualPgVersion = do
             , matchHeaders = [ "Location" <:> "/compound_pk_view?k1=eq.1&k2=eq.test"
                              , "Content-Range" <:> "*/*" ]
             }
+
+      it "should not throw and return location header when a PK is null" $
```

Comment:
I'm really puzzled as to how a PK can be null. Perhaps we can add a short comment of how that happens?

Reply:
For instance, in the use case exemplified in this test, PostgREST is recognizing both the `id` and `sponsor_id` columns of the VIEW as PRIMARY KEYS. In this case, a `competitor` without a `sponsor` is possible, this means that the column `sponsor_id` (identified as a PK in the VIEW) could have a NULL value.

With this regression fix, at the moment the Location Header is built, PostgREST handles this NULL value for the `sponsor_id` and returns the header correctly.

Reply:
Cool. So it's the inferred PKs on the VIEW that are causing the issue - 2 PKs for this view, and one of them is null.